source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -1054,14 +1054,22 @@ void ESolver_KS_LCAO<TK, TR>::after_scf(UnitCell& ucell, const int istep)
     ModuleBase::timer::tick("ESolver_KS_LCAO", "out_deepks_labels");
 #endif
 
+    //! 10) Perform RDMFT calculations
     /******** test RDMFT *********/
     if ( PARAM.inp.rdmft == true ) // rdmft, added by jghan, 2024-10-17
```

Comment:
Note that this PR deletes some developers' names, the following code change can make this PR consistent with itself
```suggestion
    if ( PARAM.inp.rdmft == true ) // rdmft
```